thoth/metrics_exporter/jobs.py

```diff
@@ -41,7 +41,7 @@
 _LOGGER = logging.getLogger("thoth.metrics_exporter.jobs")
 
 
-def countGraphSyncJobStatus(JobListItems: dict) -> (int, int, int):
+def countGraphSyncJobStatus(JobListItems: list) -> dict:
```

Comment:
The initialization bellow can be simplified:

```python
dict.fromkeys(graph_sync_job_status, 0)
```

Not necessary to change, just nitpicking :)

Also the "else" branch might be misleading when checking status:

```python
        elif "pending" in item["status"].keys():
            graph_sync_jobs_status[job_type]["pending"] += 1
        else:
            _LOGGER.error("Unknown job status %r", ...)
```

Being explicit is better as it might uncover issues later on.